components/accordion/stories/accordion.test.js

```diff
@@ -80,9 +61,10 @@ export const longerContent = new Map([
 		"Are Adobe products worth it?",
 		{
 			content: Typography({
+				size: "s",
```

Comment:
The typography font size doesn't line up with the spec'd font size for accordion item content, so we're using a smaller typography size here.